### PR TITLE
[FIX] Removed XNAT integration (was unused)

### PR DESCRIPTION
**Pull request by DESm1th**
We havent used the XNAT integration for linked IDs since the dashboard was made, and now that we're moving studies to the KCNI server it is causing crashes so I've removed it.

### CODE REVIEW

File: bin/dm_link_shared_ids.py [hunk 1]
```diff
@@ -123,7 +96,12 @@ def get_project_redcap_records(config, redcap_cred):
...
     try:
-        project_records = parse_records(response, current_study)
+        id_map = config.get_key('ID_MAP')
+    except UndefinedSetting:
+        id_map = None
+
+    try:
+        project_records = parse_records(response, current_study, id_map)
     except ValueError as e:
         logger.error("Couldnt parse redcap records for server response {}. "
```
Comment (josephmje) [inline bin/dm_link_shared_ids.py:106]:
```suggestion
        logger.error("Couldn't parse redcap records for server response {}. "
```

**Review by josephmje — APPROVED** [review]
Thanks @DESm1th! Looking good to me. Only question I had is whether there's a reason for switching the logging from StreamHandler to basic config? I don't have any preference on this. I've just been following what was done in the past.

**Comment by DESm1th** [comment]
It's just less verbose. basicConfig just adds a streamhandler.

**Comment by jerdra** [comment]
looks much cleaner now! thanks for the refactor Dawn :1st_place_medal:

**Review by jerdra — APPROVED** [review]

**Review by josephmje — APPROVED** [review]

**Pull request merged**